### ISSUE TITLE
Test Accessibility: Product Management

### ISSUE THREAD

**Issue by naragurgel** [naragurgel/serenity_oils_aroma#59]
# Steps to take

1. Open a new incognito tab in the web browser.
2. Enter the URL of the [product management](https://serenity-oils-3beb26e98281.herokuapp.com/products/add/)
3. Right-click anywhere on the page and select "Inspect" to open the browser's developer tools.
4. In the developer tools panel, navigate to the "Audits" tab.
5. Click on the "Run audits" button to initiate the Lighthouse audit.

# Expected Outcome

- [ ]  Accessibility should be 90+
